code_of_conduct.html

```diff
@@ -77,74 +77,62 @@
+
+    <p>If you feel that the Code of Conduct has been violated, you can submit a report via the <a href="https://numfocus.typeform.com/to/ynjGdT?typeform-source=numfocus.org">NumFOCUS Code of Conduct Reporting Form</a>.
 
+    <h2>Who Will Receive Your Report</h2>
+
+    <p>Your report will be received and handled by NumFOCUS Code of Conduct Working Group;
+    trained, and experienced contributors with diverse backgrounds. The group is making
+    decisions independently from the project, PyData, NumFOCUS, or any other organization.</p>
+
+    <p>The Working Group will work with the Astropy Project's Ombudsperson to resolve an incident:
+    The NumFOCUS Code of Conduct Working group will review the incident, and provide recommendations on how to handle this or what consequences or sanction might be appropriate. As per <a href="https://github.com/astropy/astropy-APEs/blob/main/APE0.rst">Astropy's governance charter</a>, the Astropy Ombudsperson along with the Coordination Committee will receive those recommendations and perform any actions necessary to address the concern.</p>
+
+    <h2>Examples</h2>
+
+	<p>To make this Code of Conduct more concrete, we provide here some hypothetical examples of how a Code of Conduct issue might arise that may be particular to our community.</p>
+
+	<ul>
+		
+	<li>A member of the Astropy community might express a preference on an online forum for a specific tool or programming language (e.g. Python) over another language. If this preference is expressed as a personal preference or with reference to particular technical merits of that language vs others, there is no violation of the Code of Conduct.  However, if that member instead expresses this preference by way of insult to those who use another language, or via violent imagery directed at those other languages or its users, that would be an act of "tool shaming" and be a violation of the Code of Conduct</li>
+
+	<li>If a member of the community knowingly uses a software tool or astronomical dataset in a public package or academic publication without acknowledging or citing the tool in a reasonable way requested by the upstream tool, this is a violation of the Code of Conduct. If the member makes a reasonable effort to find an acknowledgement and one is not available, this would not be a Code of Conduct violation (although it might or might not represent a violation of copyright law depending on the details of the situation and adopted license).</li>
+
+	<li>If an Astropy maintainer were to post a message in a public forum that is insulting an astronomy research community member's skills as a programmer, this would be a violation of the Code of Conduct, as the researcher is also a member of our community. If that same thing were stated in a meeting with other Astropy maintainers, it may or may not represent a code of conduct violation, depending on whether the intent is to insult vs pointing out a skill lack in the community in a productive manner.</li>
```

Comment:
If you must include this one, I feel like we should also include one that goes another way; for example, a researcher that rudely dismisses a code review from a programmer on the basis that the latter is not as wise in the way of science as the former, even thought the review comments are valid. 

```suggestion
	<li>If an Astropy maintainer were to post a message in a public forum that is insulting an astronomy research community member's skills as a programmer, this would be a violation of the Code of Conduct, as the researcher is also a member of our community. If that same thing were stated in a meeting with other Astropy maintainers, it may or may not represent a code of conduct violation, depending on whether the intent is to insult v.s. pointing out a skill lack in the community in a productive manner.</li>
```

Reply:
"vs." is fine.

But more importantly, the following comment remains unaddressed: "I feel like we should also include one that goes another way..."